### PR TITLE
webui: Updated localization

### PR DESCRIPTION
**Pull request by fbergkemper**
#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

##### General

- [x] PR name is meaningful
- [x] Purpose of the PR is understood
- [x] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same
- [x] Commit descriptions are understandable and well formatted

### CODE REVIEW

File: webui/module/Application/language/de_DE.po [hunk 1]
```diff
@@ -105,10 +105,33 @@ msgstr "Director"
...
 
+#: ../module/Application/src/Application/View/Helper/ACLAlert.php:38
+msgid "Please read the <a href=\"http://doc.bareos.org/master/html/bareos-manual-main-reference.html#sec:webui-console\" target=\"_blank\">Bareos documentation</a> for any additional information on how to configure the Command ACL directive of your Console/Profile resources. Following is a list of required commands which need to be in your Command ACL to run this module properly:"
+msgstr "Please read the <a href=\"http://doc.bareos.org/master/html/bareos-manual-main-reference.html#sec:webui-console\" target=\"_blank\">Bareos documentation</a> for any additional information on how to configure the Command ACL directive of your Console/Profile resources. Following is a list of required commands which need to be in your Command ACL to run this module properly:"
+
 #: ../module/Application/view/error/404.phtml:3
-#, fuzzy
 msgid "A 404 error occurred"
-msgstr "ein Fehler ist aufgetreten"
+msgstr "A 404 error occurred"
```
Comment (pstorz) [inline webui/module/Application/language/de_DE.po:114]:
```suggestion
msgstr "Ein Fehler ist aufgetreten (404)"
```

File: webui/module/Application/language/de_DE.po [hunk 2]
```diff
@@ -148,7 +171,7 @@ msgstr "Stack trace"
 #: ../module/Application/view/error/index.phtml:32
 #: ../module/Application/view/error/404.phtml:76
 msgid "Previous exceptions"
-msgstr "vorige Ausnahmen"
```
Comment (pstorz) [inline webui/module/Application/language/de_DE.po:151]: Why is the translation reverted?

File: webui/module/Application/language/de_DE.po [hunk 3]
```diff
@@ -1176,31 +1211,27 @@ msgid "Autochanger"
 msgstr "Medienwechsler"
 
 #: ../public/js/bootstrap-table-formatter.js:638
-#, fuzzy
 msgid "Manage autochanger"
-msgstr "Medienwechsler"
+msgstr "Manage Medienwechsler"
```
Comment (pstorz) [inline webui/module/Application/language/de_DE.po:1215]:
```suggestion
msgstr "Medienwechsler verwalten"
```

**Review by pstorz — CHANGES_REQUESTED** [review]
There are some words not translated into german. I guess this is intentionally?

File: webui/module/Application/language/de_DE.po (hunk 1 above)
Comment (fbergkemper) [reply webui/module/Application/language/de_DE.po]: Applied

File: webui/module/Application/language/de_DE.po (hunk 3 above)
Comment (fbergkemper) [reply webui/module/Application/language/de_DE.po]: Applied

File: webui/module/Application/language/de_DE.po (hunk 2 above)
Comment (fbergkemper) [reply webui/module/Application/language/de_DE.po:151]: Mistake, changed it.

**Comment by fbergkemper** [comment]
> 
> 
> There are some words not translated into german. I guess this is intentionally?

Yes, mostly those ones are error message, which we may or may not translate in future.

**Review by pstorz — APPROVED** [review]
Looks good.

**Pull request merged**